README.md

```diff
@@ -131,8 +131,12 @@ nxp.betweenness_centrality(H)
 
    ```py
    # using `name` parameter - nx-parallel as an independent package
-   nxp.all_pairs_node_connectivity(H) # runs the parallel implementation in `connectivity/connectivity`
-   nxp.approximate_all_pairs_node_connectivity(H) # runs the parallel implementation in `approximation/connectivity`
+   nxp.all_pairs_node_connectivity(
+       H
+   )  # runs the parallel implementation in `connectivity/connectivity`
```

Comment:
The linter will avoid splitting this over multiple lines if you put the comment on its own line before the command ratherthan at the ened of the command. It is a style choice so not always better, but often it is.  Does the result look better to you?
```suggestion
   # runs the parallel implementation in `connectivity/connectivity
   nxp.all_pairs_node_connectivity(H)
```

And similarly for the lines below.